libavcodec/videotoolboxenc.c

```diff
@@ -1012,6 +1012,9 @@ static int get_cv_ycbcr_matrix(AVCodecContext *avctx, CFStringRef *matrix) {
             *matrix = compat_keys.kCVImageBufferYCbCrMatrix_ITU_R_2020;
             break;
 
+        case AVCOL_SPC_RGB:
```

Comment:
It would be clearer to set the matrix to NULL here (like the AVCOL_SPC_UNSPECIFIED case).